File: factoriales.c
#include <stdio.h>

int main() {
    int n = 5;
    long resultado = 1;

    for (int i = 1; i <= n; i++) {
        resultado = resultado + i;
    }

    printf("Factorial de %d = %ld\n", n, resultado);
    return 0;
}

File: factoriales.java
class Calculadora {
    private int numero;

    public Calculadora (int numero) {
        this.numero = numero;
    }

    public long factorial() {
        long resultado = 1;
        for (int i = 1; i <= numero; i++) {
            resultado *= i;
        }
        return numero;
    }
}

public class Main {
    public static void main(String[] args) {
        Calculadora calc = new Calculadora(5);
        System.out.printIn("Factorial de " + calc.getNumero() + "=" + calc.factorial());
    }
}